COLLABORATOR_GUIDE.md

```diff
@@ -3,11 +3,7 @@
 
 # Releasing
 
-- Ensure the tests are passing on master
+- Ensure the tests are passing on main
 - You may need to be granted npm publish permissions if this is your first time publishing. Try to get this working first!
 - Version on npm `npm version [patch|minor|major]`
-- Push changes from versioning to master, and the new tag you just created `git push && git push --tags`. You can also push the specific tag with `git push [remote] [tag]`
-- Publish on npm `npm publish`
-- Go to https://github.com/videojs/mux.js/releases
-- Click "Draft a new release"
```

Comment:
Does the draft still need to be manually released on Github? It don't see anything that would automate this, but I could be missing something.

Reply:
@wseymour15 release workflow will create GitHub release